Tests/Fixtures/Generated/XCAssets/swift3-context-all-customname.swift

```diff
@@ -17,6 +17,93 @@ internal typealias XCTData = NSDataAsset
 // swiftlint:disable superfluous_disable_command
 // swiftlint:disable file_length
 
+// MARK: - Asset Catalogs
+
+// swiftlint:disable identifier_name line_length nesting type_body_length type_name
+internal enum XCTAssets {
+  internal enum Colors {
+    internal enum _24Vision {
+      internal static let background = XCTColorAsset(name: "24Vision/Background")
+      internal static let primary = XCTColorAsset(name: "24Vision/Primary")
+    }
+    internal static let orange = XCTImageAsset(name: "Orange")
+    internal enum Vengo {
+      internal static let primary = XCTColorAsset(name: "Vengo/Primary")
+      internal static let tint = XCTColorAsset(name: "Vengo/Tint")
+    }
+    // swiftlint:disable trailing_comma
+    internal static let allColors: [XCTColorAsset] = [
+      _24Vision.background,
+      _24Vision.primary,
+      Vengo.primary,
+      Vengo.tint,
+    ]
+    internal static let allDataAssets: [XCTDataAsset] = [
+    ]
+    internal static let allImages: [XCTImageAsset] = [
+      orange,
+    ]
+    // swiftlint:enable trailing_comma
+    @available(*, deprecated, renamed: "allImages")
+    internal static let allValues: [XCTAssetsType] = allImages
```

Comment:
⚠️ This is gonna conflict with the other PR removing that deprecated property, we should be careful when we're gonna rebase

Reply:
No problem 😄